modules/nft/client/cli/tx.go

```diff
@@ -303,3 +324,43 @@ func GetCmdBurnNFT() *cobra.Command {
 
 	return cmd
 }
+
+// GetCmdTransferDenom is the CLI command for sending a TransferDenom transaction
+func GetCmdTransferDenom() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  "transfer-denom <denom-id> <recipient>",
```

Comment:
```suggestion
		Use:  "transfer-denom [recipient] [denom-id]",
```

All sending/transfer-related tx keep the same custom `tx send/transfer [from/sender] [to/recipient] [token/nft]`, `from/sender` is **optional**.

Reply:
Update this command args.